server/data/licenceApiClient.ts

```diff
@@ -109,8 +109,8 @@ export default class LicenceApiClient extends RestClient {
     )
   }
 
-  async updateLicenceStatus(licenceId: string, statusRequest: StatusUpdateRequest, user: User): Promise<void> {
-    await this.put({ path: `/licence/id/${licenceId}/status`, data: statusRequest }, { username: user.username })
+  async updateLicenceStatus(licenceId: string, statusRequest: StatusUpdateRequest, user?: User): Promise<void> {
+    await this.put({ path: `/licence/id/${licenceId}/status`, data: statusRequest }, { username: user?.username })
   }
```

Comment:
Is this so it can be called from a user-less session i.e. the queue listener?

Reply:
Yes, the actions taken by the system will not have a user associated, so the system token is used